src/main/java/com/three_tech_solutions/slot_app/data/repositories/StudentRepository.java

```diff
@@ -21,7 +21,7 @@ OR LOWER(CONCAT(s.name, ' ', s.lastname)) LIKE LOWER(CONCAT('%', :filter, '%'))
            OR LOWER(CONCAT(s.lastname, ' ', s.name)) LIKE LOWER(CONCAT('%', :filter, '%'))
            OR s.dni LIKE CONCAT(:filter, '%'))
     """)
-    Page<Student> getStudentsByUserAndNameAndLastnameAndDni(@Param("user") User user, @Param("filter") String filter, Pageable pageable);
+    Page<Student> getStudentsByUserAndNameAndLastnameAndDni(@Param("user") User user, @Param("filter") String filter, Pageable pageableWithSort );
```

Comment:
Acá no hace falta que le cambiemos el nombre y específiquemos que tiene un ordenamiento, con pageable ya es suficiente ya que esta es una interfaz propia de Spring que representa a la paginacion y ordenamiento

```suggestion
    Page<Student> getStudentsByUserAndNameAndLastnameAndDni(@Param("user") User user, @Param("filter") String filter, Pageable pageable);
```